src/plugin.js

```diff
@@ -162,7 +162,6 @@ class PlaylistMenuItem extends Component {
 
     nowPlayingEl.className = 'vjs-playlist-now-playing-text';
     nowPlayingEl.appendChild(document.createTextNode(nowPlayingText));
-    nowPlayingEl.setAttribute('title', nowPlayingText);
```

Comment:
These `title` attributes were not necessary as they repeated the text content in the element, which is not a best practice for accessibility.

Reply:
I believe the reason for this `title` attribute is so that if you have a title that's too long that gets cut off, if you hover over it, it would be readable still. Making it so that the screen reader doesn't read it twice is a good point, though.

Reply:
I think I'd prefer if we keep it for now but open an issue/schedule work to come up with an accessible solution.

Reply:
Fair enough, will keep it.